### DISCUSSION TITLE
카메라로 사진을 찍으면 캐릭터가 머리카락과 분리되는 이유가 뭘까요.ㅠ

### DISCUSSION THREAD

**Discussion by diamondshop7** [naverz/zepeto-studio-kor#1253] [5. Q&A]
카메라로 사진을 찍었을 때 캐릭터 머리카락과 몸이 분리되어 대머리처럼 되고 다리가 땅에 박힌 채 사진이 찍힙니다.
또 캐릭터 이동버튼과 사진 자동저장 버튼이 맞물려 저장이 되지 않습니다.
어떻게 해야 해결할 수 있을까요?

**Comment by chlguswls32** [comment]
글로는 내용 파악이 어려우니, 스크린샷 같이 올려주시면 답변드리는데 도움될것같아요!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 캐릭터와 몸이 분리되는 현상과 관련하여 https://github.com/naverz/zepeto-studio-kor/discussions/452 이슈와 같은 현상인지 확인 부탁 드립니다.
캐릭터 이동 버튼과 카메라 저장 버튼은 해상도 설정을 맞춰보신 후 테스트 해보시기를 권장 드리며
그래도 겹칠 경우 v-pad prefab을 열어서 이동 버튼 위치 및 크기 조절이 가능합니다.
<img width="1500" alt="Screen Shot 2022-09-05 at 10 03 53 AM" src="https://user-images.githubusercontent.com/100118650/188341731-76023260-1be9-49b2-bc54-e9f1cecd07a2.png">

감사합니다.

**Comment by diamondshop7** [comment]
![7](https://user-images.githubusercontent.com/111335692/188361742-5d1cebf0-f20b-4433-bbb2-603328187234.png)
이렇게 머리가 들려요.ㅠㅠ

**Reply by Zepeto-TechEvangelist** [reply]
#452 이슈와 동일한 증상으로 아래 커밋로그를 참고하셔서 해당 코드 수정을 하시면 정상적으로 보여질 것입니다 :)
https://github.com/naverz/zepeto-multiplay-example/commit/0fb211dde49a186e1ebd9782e5a0b70c1b0a6664
